drivers/i2s/i2s_cavs.c

```diff
@@ -642,6 +642,11 @@ static int i2s_cavs_trigger(const struct device *dev, enum i2s_dir dir,
 	unsigned int key;
 	int ret = 0;
 
+	if (dir == I2S_DIR_BOTH) {
+		LOG_ERR("I2S_DIR_BOTH value is not supported");
+		return -ENOSYS;
```

Comment:
When looking at the #33690 that unified usage of ENOSYS/ENOTSUP drivers generally don't print error message when they return ENOSYS. The error code is well defined. We could save some bytes in flash if we remove `LOG_ERR`.

If we leave the error message we should probably change it to "I2S_DIR_BOTH value is not implemented". Since every driver that supports RX and TX can potentially support I2S_DIR_BOTH.

Reply:
> The error code is well defined. We could save some bytes in flash if we remove `LOG_ERR`.

Right. I'll remove these messages.